client/src/main/scala/drt/client/services/JSDateConversions.scala

```diff
@@ -68,6 +68,8 @@ object JSDateConversions {
       def getZone(): String = date.tz()
 
       override def getTimeZoneOffsetMillis(): MillisSinceEpoch = date.utcOffset().toLong * 60000L
+
+      def startOfTheMonth(): SDateLike = SDate(date.getFullYear(), date.getMonth(), date.getDate(), 0, 0)
```

Comment:
is this timezone safe?

Reply:
yes, the SDate apply method handles the time zone